examples/big-bang/zarf.yaml

```diff
@@ -13,7 +13,7 @@ components:
       - name: flux-installer
         # This will be built on the package create side and deployed as a regular manifest on package deploy
         kustomizations:
-          - https://repo1.dso.mil/platform-one/big-bang/bigbang.git//base/flux?ref=1.17.0
+          - https://repo1.dso.mil/platform-one/big-bang/bigbang.git//base/flux?ref=1.27.1
```

Comment:
I think with a version change this big there are going to be changes needed in the `kustomizations/bigbang/values.yaml` that are going to need to accompany this. I suspect the helm charts for some of these repos have changed and are going to need their keys and values updated to get the BB package to deploy.